classes/DepositPackage.inc.php

```diff
@@ -429,83 +430,85 @@ public function transferDeposit() {
 		$plnPlugin = PluginRegistry::getPlugin('generic',PLN_PLUGIN_NAME);
 
 		// post the atom document
-		$url = $plnPlugin->getSetting($journalId, 'pln_network');
+		$baseUrl = $plnPlugin->getSetting($journalId, 'pln_network');
 		$atomPath = $this->getAtomDocumentPath();
 
-		if ($this->_deposit->getLockssAgreementStatus()) {
```

Comment:
Instead of using this check to decide between a "add" or "edit", I've sent a request to the `/state` route.